lstchain/calib/camera/calib.py

```diff
@@ -115,22 +113,22 @@ def lst_calibration(event, telescope_id, high_gain_threshold):
     pedcorrectedsamples = data - np.atleast_3d(ped) / nsamples
 
     integrator = extractor.LocalPeakWindowSum()
-    integration, pulse_time = integrator(pedcorrectedsamples)  # these are 2D matrices num_gains * num_pixels
+    integration, peak_time = integrator(pedcorrectedsamples)  # these are 2D matrices num_gains * num_pixels
 
     signals = integration.astype(float)
 
     dc2pe = event.mc.tel[telescope_id].dc_to_pe  # numgains * numpixels
     signals *= dc2pe
 
     # threshold = 4094
-    image, pulse_time = gain_selection(data, signals, pulse_time, high_gain_threshold)
+    image, peak_time = gain_selection(data, signals, peak_time, high_gain_threshold)
 
     event.dl1.tel[telescope_id].image = image
-    event.dl1.tel[telescope_id].pulse_time = pulse_time
+    event.dl1.tel[telescope_id].peak_time = peak_time
 
 
 @deprecated('28/06/2019', message='gain selection is now performed at <= R1 calibration level')
-def gain_selection(waveform, charges, pulse_time, threshold):
+def gain_selection(waveform, charges, peak_time, threshold):
```

Comment:
shall we get rid of the deprecated functions already?

Reply:
Can I do it in a subsequent PR?

Reply:
sure, probably better